README.md

```diff
@@ -621,6 +621,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [gopsutil](https://github.com/shirou/gopsutil) - A cross-platform library for retrieving process and system utilization(CPU, Memory, Disks, etc).
 * [gosms](https://github.com/haxpax/gosms) - Your own local SMS gateway in Go that can be used to send SMS
 * [gountries](https://github.com/pariz/gountries) - A package that exposes country and subdivision data.
+* [hanu](https://github.com/sbstjn/hanu) - Framework for writing Slack bots. [How to host your Slack bot on Heroku with hanu …](https://sbstjn.com/host-golang-slackbot-on-heroku-with-hanu.html)
```

Comment:
There are a limit of links per list item in our guidelines. Please remove the second link.